shpool/tests/attach.rs

```diff
@@ -1433,6 +1433,51 @@ fn motd_debounced_pager_unbounces() -> anyhow::Result<()> {
     })
 }
 
+#[test]
+#[timeout(30000)]
+fn motd_env_test_pager() -> anyhow::Result<()> {
```

Comment:
nit: should we call is `modt_env_test_pager_preserves_term_env_var` or similar since i think that is principally what is is testing?

Reply:
Yeah good idea, thanks!